CHANGES.md

```diff
@@ -1,5 +1,32 @@
 # Changes
 
+## edge-22.8.1
+
+This releases introduces default probe authorization. This means that on
+clusters that use a default `deny` policy, probes do not have to be explicitly
+authorized using policy resources. Additionally, the
+`policyController.probeNetworks` value has been added which allows users to
+configure the networks that probes are expected to be performed from.
+
+Additionally, the `linkerd authz` command has been updated to support the policy
+resources AuthorizationPolicy and HttpRoute.
+
+Finally, some smaller changes include allowing to disable `linkerd-await` on
```

Comment:
grammar nit:
```suggestion
Finally, some smaller changes include allowing disabling `linkerd-await` on
```

Reply:
I think "to disable" still reads better, but maybe I can change the wording in general.